pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
 dependencies = [
   "jupyter_server>=1.6,<3",
   "jupyterlab_git",
-  "jupyter_ai>=3.0.0-beta.1"
+  "jupyter_server_documents>=0.1.a8"
```

Comment:
Is jupyter_server_documents a requirement?

Reply:
Yes, the `ydoc` is being pulled from `yroom_manager`. 
https://github.com/jupyter-ai-contrib/jupyter-ai-tools/blob/main/jupyter_ai_tools/utils.py#L69-L85

We can make it compatible with collaboration with a small change, let me update this.